src/rules/missing-playwright-await.test.ts

```diff
@@ -368,5 +383,14 @@ runRuleTester('missing-playwright-await', rule, {
         },
       },
     },
+    // Page methods
+    { code: `await page.goto('https://example.com')` },
+    { code: `await page.title()` },
+    // Other page methods are ignored
+    { code: `page.frames()` },
+    // Other methods with the same name are ignored
+    { code: `randomObject.title()` },
+    // Does not need to be awaited when returned
+    { code: `() => { return page.content() }` },
```

Comment:
Let's add a few tests for `this`, e.g. `this.page.goto()`

Reply:
Also, this probably doesn't support stuff like this: `page.locator('.foo').click()`.  Without that, this rule is pretty limited in usefulness.